doc/source/whatsnew/v0.16.1.txt

```diff
@@ -108,6 +108,7 @@ Bug Fixes
 - Bug in plotting continuously using ``secondary_y`` may not show legend properly. (:issue:`9610`, :issue:`9779`)
 
 - Bug in ``DataFrame.plot(kind="hist")`` results in ``TypeError`` when ``DataFrame`` contains non-numeric columns  (:issue:`9853`)
+- Bug in continuous ``DataFrame`` with ``DatatimeIndex`` plot may raise ``TypeError`` (:issue:`9852`)
```

Comment:
I would describe this as `Bug where repeated plotting of DataFrame with a DatetimeIndex may raise TypeError`